bin/build.sh

```diff
@@ -451,4 +454,17 @@ then
   puts "Regression tests complete"
 fi
 
+if [ ${INSTALL_OBJECTIVEC} -eq 1 ]
+then
+  puts "Installing GNU Objective-C for linux"
+
+  sudo apt-get -y install gobjc gnustep gnustep-make gnustep-common gnustep-devel
```

Comment:
I think we'd probably want to rewrite this line like this https://github.com/smackers/smack/blob/develop/bin/build.sh#L187.